### PR TITLE
Update workflows to support the removal of the bundle dir

### PR DESCRIPTION
**Pull request by bcrochet**
In order to remove the bundle dir from the repo, we need to have
some additional actions occur.

* Run a 'make bundle'
* Add the required labels to annotations.yaml and bundle.Dockerfile
* Make IMAGE_REGISTRY a secret, so that we can test in forks

Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -9,50 +9,68 @@ jobs:
...
+        RELEASE_TAG=${{ env.RELEASE_TAG }} \
+        VERSION=${{ env.RELEASE_TAG }} \
+        USE_IMAGE_DIGESTS=true \
+        make bundle
+
+    # This is a very naive implementation. Mainly for PoC
+    - name: Add OpenShift metadata to bundle
+      id: add-openshift-metadata
+      run: |
+        echo '  com.redhat.openshift.versions: "v4.8-v4.10"' >> bundle/annotations.yaml
```
Comment (acornett21) [inline .github/workflows/build-release.yml:55]: This is a TAB right? So that the YAML is indented properly correct?

**Review by acornett21 — CHANGES_REQUESTED** [review]

File: .github/workflows/build-release.yml [hunk 2]
```diff
@@ -9,50 +9,68 @@ jobs:
...
-    env:
-      IMAGE_REGISTRY: quay.io/opdev
     steps:
     - uses: actions/checkout@v2
     - name: Set Env Tags
-      run: echo RELEASE_TAG=$(echo $GITHUB_REF | cut -d '/' -f 3) >> $GITHUB_ENV
+      run: |
+        echo RELEASE_TAG=$(echo $GITHUB_REF | cut -d '/' -f 3) >> $GITHUB_ENV
+        echo IMAGE_REGISTRY=$(echo ${{ secrets.IMAGE_REGISTRY }} | cut -d '/' -f 1) >> $GITHUB_ENV
+        echo IMAGE_REPO=$(echo ${{ secrets.IMAGE_REGISTRY }} | cut -d '/' -f 2) >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (bcrochet) [reply .github/workflows/build-release.yml:18]: Not copy pasta, but not thinking. Fixing.

[the remark this answers is not in the document]
Comment (bcrochet) [reply .github/workflows/build-release.yml:18]: No. That's actually correct. IMAGE_REGISTRY secret contains 'quay.io/repo'

File: .github/workflows/build-release.yml (hunk 1 above)
Comment (bcrochet) [reply .github/workflows/build-release.yml:55]: Not a tab... DIdn't think it was a tab.

Comment (bcrochet) [reply .github/workflows/build-release.yml:55]: They're spaces in the generated file.

20 20 is two spaces in the first line. And the last.

```
00000060  72 65 67 69 73 74 72 79  2b 76 31 0a 20 20 6f 70  |registry+v1.  op|
00000070  65 72 61 74 6f 72 73 2e  6f 70 65 72 61 74 6f 72  |erators.operator|
00000080  66 72 61 6d 65 77 6f 72  6b 2e 69 6f 2e 62 75 6e  |framework.io.bun|
00000090  64 6c 65 2e 6d 61 6e 69  66 65 73 74 73 2e 76 31  |dle.manifests.v1|
000000a0  3a 20 6d 61 6e 69 66 65  73 74 73 2f 0a 20 20 6f  |: manifests/.  o|
```

**Review by acornett21 — APPROVED** [review]

**Review by jomkz — APPROVED** [review]

**Pull request merged**